core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,58 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
+      UPDATE
+        spree_adjustments
+      SET
+        order_id = adjustable_id
+      WHERE
+        adjustable_type = 'Spree::Order'
+      ;
+    SQL
+
+    execute(<<-SQL.squish)
+      UPDATE
+        spree_adjustments
+      SET
+        order_id =
+          (SELECT order_id FROM spree_line_items WHERE spree_line_items.id = spree_adjustments.adjustable_id)
+      WHERE
+        adjustable_type = 'Spree::LineItem'
```

Comment:
Sure. Will do. I just checked our internal migration and it also had this clause. It was dropped for spree upstreaming for some reason